src/server/cloak/index.js

```diff
@@ -90,7 +89,16 @@ module.exports = (function() {
         delete config.express;
       }
 
-      io.set('log level', config.logLevel);
+      // Cloak defaults Socket.IO log level to 1
+      io.set('log level', 1);
```

Comment:
Users may be setting the `logLevel` option today, making this a breaking change. If you want to keep the same major version for the next release, you should write:

```js
// Maintain `logLevel` option for backwards compatibility.
// TODO: Remove this in next major release
if ('logLevel' in config) {
  if ('log level' in ioConfig) {
    throw new Error(
      'Socket.io logging level over-specified. Use config.logLevel ' +
      'or config["socket.io"]["log level"] (not both)'
    );
  }
  ioConfig['log level'] = config.logLevel;
}
```